### PR TITLE
EASE-85 adds enable and disable module toggle

### PR DESCRIPTION
**Pull request by danvaida**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -4,6 +4,7 @@ provider "aws" {
 }
 
 resource "aws_iam_user" "user" {
```
Comment (maartenvanderhoef) [inline main.tf]: Please rename the `"user"`  `"this"` to keep that output paradigm with `this_user_arn` etc. tip top.

File: main.tf [hunk 2]
```diff
@@ -12,6 +13,7 @@ resource "aws_iam_user" "user" {
 }
 
 resource "aws_iam_access_key" "access_key" {
```
Comment (maartenvanderhoef) [inline main.tf]: same here, access_key - > this

File: outputs.tf [hunk 3]
```diff
@@ -1,20 +1,20 @@
 output "this_user_name" {
-  value       = "${aws_iam_user.user.name}"
+  value       = "${element(concat(aws_iam_user.user.*.name, list("")), 0)}"
```
Comment (maartenvanderhoef) [inline outputs.tf]: splat operator love <3

**Comment by maartenvanderhoef** [comment]
It's out of scope now I think, but for a module to be a bit more generic it would be good to support both `secret` and `encrypted_secret`.

File: main.tf (hunk 1 above)
Comment (danvaida) [reply main.tf]: nice catch! done.

**Comment by danvaida** [comment]
> It's out of scope now I think, but for a module to be a bit more generic it would be good to support both `secret` and `encrypted_secret`.

Yep, we should handle that in a separate PR.

**Pull request merged**